platform/posix/setup_cantera.csh.in

```diff
@@ -1,4 +1,8 @@
 #!/bin/csh
+printf "%s\n" \
+    "The setup_cantera script is deprecated and will be removed after Cantera 2.6." \
+    "If you use this script, please leave a comment at:" \
+    "https://github.com/Cantera/enhancements/issues/135"
```

Comment:
I actually doubt anyone is using the `PYTHONPATH` anyways... I'd prefer to let them comment in the GH issue and direct them to environments there, especially because this file is much more than `PYTHONPATH`

Reply:
In other words, I want to keep the message here short so people read it 😄

Reply:
ok. I used `setup_cantera` once upon a time to adjust `PYTHONPATH` (pre-conda). But even if you want to keep it short, I'd suggest changing `please leave a comment at:` to `find information on migration at:` as I do not think there's a reason to have a reversion plan.